specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/multivariate/models.cadl

```diff
@@ -0,0 +1,360 @@
+  changedVariables?: string[];
+}
+
+@doc("""
+Training result of a model including its status, errors and diagnostics
+information.
+""")
+model ModelInfo {
+  @doc("""
+Source link to the input data to indicate an accessible Azure storage Uri,
+either pointed to an Azure blob storage folder, or pointed to a CSV file in
+Azure blob storage based on you data schema selection. 
+""")
+  dataSource: string;
+
+  @doc("""
+Data schema of input data source: OneTable or MultiTable. The default
+DataSchema is OneTable.
+""")
+  dataSchema?: DataSchema;
+
+  @doc("""
+A required field, indicating the start time of training data, which should be
+date-time of ISO 8601 format.
+""")
+  startTime: zonedDateTime;
+
+  @doc("""
+A required field, indicating the end time of training data, which should be
+date-time of ISO 8601 format.
+""")
+  endTime: zonedDateTime;
+
+  @doc("""
+An optional field. The display name of the model whose maximum length is 24
+characters.
+""")
+  @maxLength(24)
+  displayName?: string;
+
+  @doc("""
+An optional field, indicating how many previous timestamps will be used to
+detect whether the timestamp is anomaly or not. 
+""")
+  slidingWindow?: int32;
+
+  @doc("An optional field, indicating the manner to align multiple variables.")
+  alignPolicy?: AlignPolicy;
+
+  @doc("Model status. One of CREATED, RUNNING, READY, and FAILED.")
+  status?: ModelStatus;
+
+  @doc("Error messages when failed to create a model.")
+  errors?: ErrorResponse[];
+
+  @doc("Diagnostics information to help inspect the states of model or variable.")
+  diagnosticsInfo?: DiagnosticsInfo;
+}
+
+@doc("An optional field, indicating the manner to align multiple variables.")
+model AlignPolicy {
+  @doc("""
+An optional field, indicating how to align different variables to the same
+time-range. Either Inner or Outer. 
+""")
+  alignMode?: AlignMode;
+
+  @doc("""
+An optional field, indicating how missing values will be filled. One of
+Previous, Subsequent, Linear, Zero, Fixed.
+""")
+  fillNAMethod?: FillNAMethod;
+
+  @doc("An optional field. Required when fillNAMethod is Fixed.")
+  paddingValue?: float32;
+}
+
+@doc("Diagnostics information to help inspect the states of model or variable.")
+model DiagnosticsInfo {
+  modelState?: ModelState;
+  variableStates?: VariableState[];
+}
+
+model ModelState {
+  @doc("""
+This indicates the number of passes of the entire training dataset the
+algorithm has completed.
+""")
+  epochIds?: int32[];
+
+  @doc("""
+List of metrics used to assess how the model fits the training data for each
+epoch.
+""")
+  trainLosses?: float32[];
+
+  @doc("""
+List of metrics used to assess how the model fits the validation set for each
+epoch.
+""")
+  validationLosses?: float32[];
+
+  @doc("Latency for each epoch. ")
+  latenciesInSeconds?: float32[];
+}
+
+@doc("Response of getting a model.")
+@resource("multivariate/models")
+model Model {
```

Comment:
Is this name too generic?

Reply:
Let's do naming discussions in the archboard :)

Reply:
Agree with Laurent, these naming updates should be done after the archboard.